### PR TITLE
Implement Network Fee row for upcoming Transaction Details sheet

### PR DESCRIPTION
**Pull request by jkadamczyk**
## What changed (plus any additional context for devs)

Implemented the UI for the network fee row + handling for fees that we get from Zerion (we weren't using them previously)

The underlying data part of this fee row still needs a bunch of work. I'm deferring that for later. I will first to figure out the UI part of this sheet.

### CODE REVIEW

File: src/hooks/useAccountProfile.ts [hunk 1]
```diff
@@ -34,12 +34,12 @@ const buildAccountProfile = (
   );
 };
 
-export default function useAccountProfile() {
-  const accountProfileSelector = createSelector(
-    [walletSelector, settingsSelector],
-    buildAccountProfile
-  );
+const accountProfileSelector = createSelector(
```
Comment (jkadamczyk) [inline src/hooks/useAccountProfile.ts:37]: Took it out of the hook, because if the selector logic doesn't depend on any data from outside redux or from the components it is used in, there is no need to create multiple instances of that selector with reselect

File: src/screens/transaction-details/components/TransactionDetailsContent.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  const formattedEthFee = getFormattedEthFee(weiFee ?? 0);
+
+  return (
+    <Box
+      background="surfacePrimary"
+      flexGrow={1}
+      paddingHorizontal="20px"
+      paddingBottom="20px"
+    >
+      {fromCurrentAddress && weiFee && (
```
Comment (skylarbarrera) [inline src/screens/transaction-details/components/TransactionDetailsContent.tsx:38]: this feels weird to me, I ~think these should be wrapped in their own typed components, `TxDetailsFeeRow`and `TxDetailsHashRow` , not really sure what else will be needed but just a thought. 

my main smell I dont like here is that were checking hash vs fee and then showing content based on those, is this gonna grow to more types or are these interchangeable ?

File: src/screens/transaction-details/helpers/getFormattedEthFee.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import { ethUnits } from '@/references';
+
+const _0_0001ETH_IN_WEI = 100000000000000;
+const _0_0001GWEI_IN_WEI = 100000;
+
+export function getFormattedEthFee(weiFee: number): string {
```
Comment (skylarbarrera) [inline src/screens/transaction-details/helpers/getFormattedEthFee.ts:6]: this is super hard to follow, im also wondering if we dont already have a helper for this?

File: src/screens/transaction-details/helpers/__tests__/getFormattedEthFee.test.ts [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import { getFormattedEthFee } from '@/screens/transaction-details/helpers/getFormattedEthFee';
+
+describe('getFormattedEthFee transaction details sheet helper', () => {
```
Comment (estrattonbailey) [inline src/screens/transaction-details/helpers/__tests__/getFormattedEthFee.test.ts:3]: Thank youuuuu

File: src/screens/transaction-details/components/TransactionDetailsContent.tsx (hunk 2 above)
Comment (estrattonbailey) [reply src/screens/transaction-details/components/TransactionDetailsContent.tsx:38]: Hmmm, this is a good question. But if they grew to more types and we had specific typed components for each, wouldn't that grow to more code than re-using generic components with different values like we have here atm?

**Review by estrattonbailey — APPROVED** [review]
LGTM if no major concerns from anyone else

File: src/parsers/transactions.ts [hunk 5]
```diff
@@ -74,6 +76,7 @@ export const parseTransactions = async (
   network: Network,
   appended = false
 ) => {
+  console.log(JSON.stringify(transactionData));
```
Comment (benisgold) [inline src/parsers/transactions.ts:79]: i think u forgot this

File: src/screens/transaction-details/TransactionDetails.tsx [hunk 6]
```diff
@@ -24,14 +26,25 @@ type Params = {
...
+  const { colors } = useTheme();
   const { transaction } = route.params as Params;
   const { height } = useDimensions();
 
+  const hash = ethereumUtils.getHash(transaction);
+  const fee = transaction.fee;
+
   return (
-    <Container height={TRANSACTION_DETAILS_SHEET_HEIGHT} deviceHeight={height}>
+    <Container
```
Comment (benisgold) [inline src/screens/transaction-details/TransactionDetails.tsx:37]: why do you need the container here?

File: src/screens/transaction-details/components/DoubleLineTransactionDetailsRow.tsx [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+export const DoubleLineTransactionDetailsRow: React.FC<Props> = ({
+  leftComponent,
+  secondaryValue,
+  value,
+  title,
+}) => (
+  <Box paddingVertical="20px">
+    <Columns space="10px" alignVertical="center">
+      <Column width="content">{leftComponent}</Column>
+      <Stack space="10px">
```
Comment (benisgold) [inline src/screens/transaction-details/components/DoubleLineTransactionDetailsRow.tsx:20]: not anything necessarily wrong here but just wondering if you know the impact of wrapping a component inside `<Columns>` with `<Column></Column>` or not?

File: src/screens/transaction-details/components/TransactionDetailsHashAndActionsSection.tsx [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+export const TransactionDetailsHashAndActionsSection: React.FC<Props> = ({
+  hash,
+}) => {
+  const formattedHash = shortenTxHashString(hash);
+  return (
+    <>
+      <TransactionDetailsDivider />
+      {formattedHash && (
+        <SingleLineTransactionDetailsRow
+          icon={'􀆃'}
```
Comment (benisgold) [inline src/screens/transaction-details/components/TransactionDetailsHashAndActionsSection.tsx:17]: this doesn't matter at all but these strings don't need to be wrapped in `{ }`

File: src/screens/transaction-details/TransactionDetails.tsx (hunk 6 above)
Comment (jkadamczyk) [reply src/screens/transaction-details/TransactionDetails.tsx:37]: I think the sheet was acting weird when there was no container. I would appreciate any tips related to using sheets. For now I just set it up so that I can implement the separate components and was planning to figure out proper height calculations at the end when the whole UI is implemented.

File: src/screens/transaction-details/components/DoubleLineTransactionDetailsRow.tsx (hunk 7 above)
Comment (jkadamczyk) [reply src/screens/transaction-details/components/DoubleLineTransactionDetailsRow.tsx:20]: Wasn't it that Column changed how the width of columns behaves? At least I derived that idea from the docs

**Review by skylarbarrera — APPROVED** [review]
